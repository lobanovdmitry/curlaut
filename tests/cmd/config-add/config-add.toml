bin.name = "curlaut"
args = "config add --alias local --realm dlobanov --username user --password password http://localhost:8080 --default --client-id dlobanov_client --client-secret ''"
fs.cwd = "fs"
fs.sandbox = true

[env.add]
HOME = "."
