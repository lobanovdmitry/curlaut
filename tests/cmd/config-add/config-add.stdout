Adding keycloak with alias: local
